docs/developers.rst

```diff
@@ -329,3 +329,108 @@ This code marks the test with `com.beam.tags.Periodic` tag. You can also specify
 You can find details about scheduling a continuous integration build under DevOps section `Configure Periodic Jobs`_.
 
 .. _Configure Periodic Jobs: http://beam.readthedocs.io/en/latest/devops.html#configure-periodic-jobs
+
+Scala tips
+^^^^^^^^^^
+Scala Collection
+~~~~~~~~~~~~~~~~
+
+Use ``mutable`` buffer instead of ``immutable var``:
```

Comment:
I was going to comment on this earlier, but waffled on it a bit due to our allocation issues - should we point out that var in general should be avoided, preferring val?

Reply:
For me it's like `common sense` when you code in Scala, that's why I did not mention it here. Should I?

Reply:
If you come from Java then it is not common sense. I can go either way, though.